src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './Tabs.scss';
+import cn from 'classnames';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: Tab,
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  return (
+    <>
+      <div className="Tabs">
+        {tabs.map(tab => (
+          <button
+            key={tab.id}
+            className={cn('Tabs__tab', {
+              'Tabs__tab--selected': tab.id === selectedTabId.id,
+            })}
+            type="button"
+            onClick={() => onTabSelected(tab)}
```

Comment:
What if the tab you're clicking on is already selected? Add this condition to decide whether you should call this callback or not